### DISCUSSION TITLE
Installation software problem on windows

### DISCUSSION THREAD

**Discussion by RoyanLiSu** [groops-devs/groops#82] [Q&A] [answered]
I want to install this software on windows. I follow step-by-step instructions for compiling and running GROOPS but in the 7th step when I want to Create the build directory and compile GROOPS with this code on MSYS2 MinGW 64-bit terminal :

mkdir /c/groops/source/build && cd /c/groops/source/build
cmake.exe .. -G "MinGW Makefiles" -DCMAKE_BUILD_TYPE=Release
mingw32-make.exe
![error](https://user-images.githubusercontent.com/110038504/181914474-fda4a7eb-0827-4435-b6f3-956107b9938f.png)

I faced with this error :
mingw32-make[2]: *** [CMakeFiles\groops.dir\build.make:76: CMakeFiles/groops.dir/base/constants.cpp.obj] Error 1
mingw32-make[1]: *** [CMakeFiles\Makefile2:84: CMakeFiles/groops.dir/all] Error 2
mingw32-make: *** [Makefile:90: all] Error 2

I attached a picture that shows the error completely. please help me to fix it.

**Comment by tmayerguerr** [comment]
Hmm, can you confirm that you are using the latest version of groops?
The problem should be solved at 25 Jun 2021 with this [commit](https://github.com/groops-devs/groops/commit/32e5fcfc4ba6347f6359ff32e604220d8bfa2926). Otherwise I have to look into the problem.

**Comment by RoyanLiSu** [comment, marked as answer]
I used the version of GROOPS that was released on 2021-09-06 and also I tried the older versions and all of them made the same error. but now I implement the last version that you put on the site three days ago, it seems very well and doesn't have any problem. I am so appreciative of your response.
